server/webapp/WEB-INF/rails/webpack/views/components/header_panel/index.js.msx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const m     = require('mithril');
+const _     = require('lodash');
+const style = require('./css/index.scss');
+
+const HeaderPanel = {
+  view(vnode) {
+    let buttons;
+    if (!_.isEmpty(vnode.attrs.buttons)) {
+      buttons = (
+        <div className={style["page-actions"] || "page-actions"}>
```

Comment:
Should we consider `camelCase` class names now for ease of use? Having to type hyphens seems too much. @naveenbhaskar may have a different opinion about this :)